course/templates/course/_course_menu.html

```diff
@@ -100,13 +100,29 @@ <h4 id="course-menu-heading">{% translate "COURSE" %}</h4>
 </li>
 {% endif %}
 
+{% for group in course_menu.student_link_groups %}
+{% if group.label == ''%}
```

Comment:
Could you add a space between `''` and `%`?

Reply:
Oh right. I can do that.